openquake/qa_tests_data/classical_damage/case_master/job_haz.ini

```diff
@@ -28,7 +28,7 @@ random_seed = 24
 truncation_level = 3
 maximum_distance = 200.0
 investigation_time = 1
-number_of_logic_tree_samples = 0
+number_of_logic_tree_samples = 1
```

Comment:
The case_master needs to have multiple branches in order to generate the statistics outputs. We should preferably keep the full enumeration.